docs/goals-canvas.md.txt

```diff
@@ -0,0 +1,26 @@
+
+
+
+| WHATS                                       | HOWS                                                         | INPUT                                         | OUTPUT                                                       | GOALS                                                        | STORIES                                                      |
+| ------------------------------------------- | ------------------------------------------------------------ | --------------------------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Filter en raadpleeg de kinderen van persoon | Door een identificatie van persoon op te geven en gevonden kind gegevens te filteren op kenmerken | BSN + optioneel naam- persoonskenmerkenfilter | (gefilterde) gegevens van kinderen zoals die bij persoon in de BRP zijn geregistreerd | - kinderen van persoon raadplegen,  - eigenschappen van kinderen raadplegen (die niet als persoon  in de BRP zijn geregistreerd),           - het BSN gebruiken om andere persoonsgegevens van kind(eren) op te vragen | [784](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/784){:target="_blank"}, [783](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/783){:target="_blank"}, [719](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/719){:target="_blank"}, [712](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/712){:target="_blank"}, [710](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/710){:target="_blank"}, [691](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/691){:target="_blank"}, [689](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/689){:target="_blank"}, [690](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/691){:target="_blank"}, [666](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/666){:target="_blank"}, [638](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/638){:target="_blank"}, [669](https://github.com/VNG-Realisatie/Haal-Centraal-BAG-bevragen/issues/669){:target="_blank"}, |
```

Comment:
Ik zou filter in de whats weghalen
Bij input bedoel je denk ik: BSN persoon + optioneel kind kenmerken filter